modules/claims_api/spec/requests/v2/veterans/rswag_disability_compensation_request_spec.rb

```diff
@@ -19,7 +19,24 @@
       produces 'application/json'
 
       get_schema_description = <<~VERBIAGE
-        The below 526 schema is in a draft state representing the attributes we are currently planning to support. Changes are expected as we continue development.#{' '}
+        <p>Automatically establishes a disability compensation claim (21-526EZ) in Veterans Benefits Management System (VBMS).#{' '}
+        This endpoint generates a filled and electronically signed 526EZ form, establishes the disability claim in VBMS, and#{' '}
+        submits the form to the Veteran's eFolder.</p>
+        <p>A 200 response indicates the API submission was successful. The claim has not reached VBMS until it has a CLAIM_RECEIVED status.#{' '}
+        Check claim status using the GET veterans/{veteranId}/claims/{id} endpoint.</p>
+        <p><strong>A substantially complete 526EZ claim must include:</strong></p>
+        <ul>
+          <li>Veteran's name</li>
+          <li>Sufficient service information for VA to verify the claimed service, if applicable</li>
+          <li>At least one claimed disability or medical condition and how it relates to service</li>
+          <li>Veteran and/or Representative signature</li>
+        </ul>
+        <p><strong>Standard and fully developed claims (FDCs)</strong></p>
+        <p><a href="https://www.va.gov/disability/how-to-file-claim/evidence-needed/fully-developed-claims/" target="_BLANK">Fully developed claims (FDCs)</a>
+        are claims certified by the submitter to include all information needed for processing. These claims process faster#{' '}
+        than claims submitted through the standard claim process. If a claim is certified for the FDC, but is missing needed information,#{' '}
+        it will be processed as a standard claim.</p>
+        <p>To certify a claim for the FDC process, set the claimProcessType to FDC_PROGRAM.</p>
       VERBIAGE
```

Comment:
When swagger tried to compile this the second to last paragraph was getting treated as just as plain text and _NOT_ wrapped in paragraph tags.  In order to bring everything in line I had to wrap it in a <p> tag, and I added that to the rest to keep everything uniform, otherwise it was just the third paragraph with <p> tags which I think would give the impression something is different with it when in reality swagger just had a hard time figuring out what to do with it for some reason.